ingestion_tools/scripts/common/image.py

```diff
@@ -353,7 +353,13 @@ def update_headers(self, mrcfile: MrcFile, header_mapper, voxel_spacing):
             if old_header.exttyp.item().decode().strip():
                 header.nsymbt = old_header.nsymbt
                 header.exttyp = old_header.exttyp
-                if ext := self.volume_reader.get_mrc_extended_header():
+                ext = self.volume_reader.get_mrc_extended_header()
+                # In order to cover the following cases:
+                # 1. do set on filled np.ndarray > np.array(val: np.ndarray).tolist() -> list
+                # 2. do set on filled np.recarray > np.array(val: np.recarray).tolist() -> list
+                # 3. skip setting on empty np.ndarray > np.array(np.empty(0)).tolist() == []
+                # 4. skip setting on None > np.array(None).tolist() == None
+                if np.array(ext).tolist():
```

Comment:
Should the headers `exttyp` also be updated accordingly?

Reply:
It is being updated in line 355, no?

Reply:
Yes, but if we are not setting the extended header in some cases where there is an extended header in the original mrc file. Should that also reflect in the `exttyp` which is currently set to the value as original mrc file.

Reply:
I see, so moving that inside of the if-clause would solve this issure, correct?

Reply:
Yeah, I believe so. One quick sanity check, what would be the expected value for that header when there is no extended_header?

Reply:
`header.exttyp = b'MRCO'`
`header.nsymbt = 0`

Reply:
Sorry, in that case, should we reset only the `nsymbt` inside the if condition?

Reply:
yep, I thought this has a default set somewhere else. I'll make the changes as requested.

Reply:
On reflection, I think it's not necessary to set the extended header type, but we can set the extended header size to zero explicitely if there is no extended header.